proposals/csharp-9.0/unconstrained-type-parameter-annotations.md

```diff
@@ -0,0 +1,78 @@
+﻿# Unconstrained type parameter annotations
+
+## Summary
+
+Allow nullable annotations for type parameters that are not constrained to value types or reference types: `T?`.
+```C#
+static T? FirstOrDefault<T>(this IEnumerable<T> collection) { ... }
+```
+
+## `?` annotation
+
+In C#8, `?` annotations could only be applied to type parameters that were explicitly constrained to value types or reference types.
+In C#9, `?` annotations can be applied to any type parameter, regardless of constraints.
+
+Unless a type parameter is explicitly constrained to value types, annotations can only be applied within a `#nullable enable` context.
```

Comment:
Occurs to me the doc doesn't state what the behavior is when the type is instantiated in an oblivious context. How does that impact the type?